Curve-finance-suite/Registry-Remove-Pool/src/agent.spec.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  HandleTransaction,
+  TransactionEvent,
+} from 'forta-agent';
+
+import { createAddress, TestTransactionEvent } from 'forta-agent-tools';
+
+import { provideHandleTransaction, REMOVE_POOL_SIGNATURE, R_IFACE } from './agent';
+
+const TARGET: string = createAddress('0xdead');
+const ALERT_ID: string = 'registry-remove-pool-test';
+const RECEIVERS: string[] = [];
+const SENDERS: string[] = [];
+
+const createFinding = ( pool: string) => Finding.fromObject({
+  name: 'Curve Registry contract called',
+  description: 'Event PoolRemoved has been emitted',
+  alertId: ALERT_ID,
+  severity: FindingSeverity.Info,
+  type: FindingType.Info,
+  protocol: 'Curve Finance',
+  metadata: {
+    pool_address: pool,
+  }
+});
+
+describe('Registry-Remove-Pool Agent tests suite', () => {
+  const handler: HandleTransaction = provideHandleTransaction(
+    ALERT_ID,
+    TARGET,
+  );
+
+  beforeAll(() => {
+    for(let i = 0; i <= 30; ++i) {
+      RECEIVERS.push(createAddress(`0xa${i}`));
+      SENDERS.push(createAddress(`0xb${i}`));
+    }
```

Comment:
Is creating so many addresses necessary? I only see one of these, `SENDERS[1]`, being used on L62.

Reply:
Having `SENDERS` and `RECEIVERS` is actually completely redundant now that I look at it. 
Both have been removed in favor of using `createAddress` for the occasional times where a different address is needed.